samples/apps/github-qna-webapp-react/README.md

```diff
@@ -8,34 +8,22 @@
 ## Running the sample
 
 1. You will need an [Open AI Key](https://openai.com/api/) or
-   [Azure Open AI Service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart)
-   for this sample.
-2. Ensure the service API is already running `http://localhost:7071`. If not learn
-   how to start it [here](../../dotnet/KernelHttpServer/README.md).
-3. **Run** the following command `yarn install` (if you have never run the sample before)
-   and/or `yarn start` from the command line.
+   [Azure Open AI Service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart) for this sample.
+2. Ensure the service API is already running `http://localhost:7071`. If not learn how to start it [here](../../dotnet/KernelHttpServer/README.md).
```

Comment:
This path for kernelhttpserver is wrong.

Reply:
hmm, weird - seems to resolve correctly on my machine. which folder does it drop you in if you click the link?

Reply:
Bah I crossed the streams, disregard.